frontend/components/side_panels/UserMenu/UserMenu.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import Icon from 'components/icons/Icon';
+import PATHS from 'router/paths';
+
+class UserMenu extends Component {
+  static propTypes = {
+    pathname: PropTypes.string,
+    onLogout: PropTypes.func,
+    onNavItemClick: PropTypes.func,
+    user: PropTypes.shape({
+      gravatarURL: PropTypes.string,
+      name: PropTypes.string,
+      username: PropTypes.string.isRequired,
+      position: PropTypes.string,
+    }).isRequired,
+  };
+
+  static defaultProps = {
+    isOpened: false,
+  };
+
+  render () {
+    const { pathname, onLogout, onNavItemClick } = this.props;
+
+    const baseClass = 'user-menu';
+    const userMenuClass = classnames(
+      baseClass,
+    );
```

Comment:
Consider making this single line
```suggestion
    const userMenuClass = classnames(baseClass);
```